amundsen_application/static/js/components/TableDetail/SourceLink/index.tsx

```diff
@@ -16,8 +21,10 @@ const SourceLink: React.FC<SourceLinkProps> = ({
 }: SourceLinkProps) => {
   if (tableSource === null || tableSource.source === null) return null;
 
-  const image =
-    tableSource.source_type === 'github' ? '/static/images/github.png' : '';
+  const image = getDescriptionSourceIconPath(tableSource.source_type);
+
+  const displayName = getDescriptionSourceDisplayName(tableSource.source_type);
```

Comment:
Nit: I would just inline these calls on the line 37 and avoid the temporal variables

Reply:
👍